docs/install.rst

```diff
@@ -226,6 +226,7 @@ Optional environment variables that can be set to configure the installation pro
 
 Possible values are given in curly brackets: {}.
 
+* ``HOROVOD_DEBUG`` - {1}. Install a debug build of Horovod with checked assertions, disabled compiler optimizations etc.
```

Comment:
is the `{1}.` a copy-paste artefact?

Reply:
@EnricoMi No, see the line above and the other environmental variables. This is the possible value for the variable.